README-pt-BR.md

```diff
@@ -0,0 +1,347 @@
+
+Se você não está familiarizado ou confiante em contribuir com o GitHub, também pode contribuir com um estudo de caso e com suas dicas e truques por meio de nosso [formulário Google](https://goo.gl/forms/akFqZEIy2kxAjfZW2).
+
+### Citando o _The Turing Way_
+
+Você pode fazer referência ao _The Turing Way_ através do arquivo Zenodo do projeto usando DOI: [10.5281/zenodo.3233853](https://doi.org/10.5281/zenodo.3233853).
+Os DOIs nos permitem arquivar o repositório e são realmente valiosos para garantir que o trabalho seja rastreado em publicações acadêmicas.
+
+A citação consistirá em algo parecido com:
+
+> The Turing Way Community, Becky Arnold, Louise Bowler, Sarah Gibson, Patricia Herterich, Rosie Higman, … Kirstie Whitaker. (2019, March 25). The Turing Way: A Handbook for Reproducible Data Science (Version v0.0.4). Zenodo. <http://doi.org/10.5281/zenodo.3233986>
+
+Você também pode compartilhar um URL mais legível que direciona a uma página do livro, por exemplo: <https://the-turing-way.netlify.com/reproducibility/03/definitions.html>, mas esteja ciente de que o projeto está em desenvolvimento, e dessa forma, esses links podem mudar com o tempo.
+Você pode querer incluir um link arquivado na web, como: <https://web.archive.org/web/20191030093753/https://the-turing-way.netlify.com/reproducibility/03/definitions.html> Assim você se certifica de não acabar com links quebrados por aí!
+
+Nós realmente apreciamos qualquer referência que você fizer ao projeto The Turing Way e esperamos que seja lhes seja útil.
+Se você tiver alguma dúvida, [entre em contato](#entre-em-contato).
+
+### Citando as ilustrações do _The Turing Way_
+
+As ilustrações do _The Turing Way_ são criadas por artistas da [Scriberia](https://www.scriberia.co.uk/) como parte do [_The Turing Way_ book dashes](https://github.com/alan-turing-institute/the-turing-way/tree/master/workshops/book-dash) em Manchester em 17 de maio de 2019 e em Londres em 28 de maio de 2019 e 21 de fevereiro de 2020.  
+Eles descrevem uma variedade de conteúdo do manual, esforços colaborativos na comunidade e o projeto _The Turing Way_ em geral.  
+Essas ilustrações estão disponíveis no Zenodo ([https://zenodo.org/record/3695300)](https://zenodo.org/record/3695300) sob uma licença CC-BY.
+
+Ao usar qualquer uma das imagens, inclua o seguinte:
+> Esta imagem foi criada pela Scriberia para a comunidade The Turing Way e é utilizada sob uma licença CC-BY.
+
+A última versão do Zenodo pode ser citada assim:
+> The Turing Way Community, & Scriberia. (2020, March 3). Illustrations from the Turing Way book dashes. Zenodo. http://doi.org/10.5281/zenodo.3695300
+
+Usamos algumas dessas ilustrações nas respostas do [Bot de boas-vindas](https://github.com/apps/welcome) às contribuições de novos membros neste repositório GitHub.
+
+### Entre em contato
+
+Temos uma sala de bate-papo no gitter e adoraríamos que você passasse lá para dizer oi acessando <https://gitter.im/alan-turing-institute/the-turing-way>.
```

Comment:
Nós temos uma sala de bate-papo no gitter e adoraríamos que você passasse lá para dar um oi! <https://gitter.im/alan-turing-institute/the-turing-way>.

Reply:
Done